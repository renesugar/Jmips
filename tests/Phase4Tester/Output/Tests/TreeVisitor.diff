--- SelfTestCases/TreeVisitor.out	2019-02-02 14:36:13.000000000 -0800
+++ ./Output/Tests/TreeVisitor.out	2019-03-13 22:31:56.000000000 -0700
@@ -1,44 +1,27 @@
-16
-100000000
-4
-8
-12
-14
-16
-20
-24
-28
-100000000
-50000000
-333
-333
-333
-28
-24
-333
-20
-16
-333
-333
-333
-14
-12
-8
-333
-4
-100000000
-1
-1
-1
-0
-1
-4
-8
-14
-16
-20
-24
-28
-0
-0
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 97 column 12: "100000000": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 170 column 12: "100000000": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 178 column 12: "50000000": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 189 column 12: "100000000": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 294 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 313 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 328 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 382 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 423 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 438 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 454 column 16: "$t1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 459 column 16: "$t2": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 460 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 513 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 642 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 670 column 16: "$s1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 708 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 719 column 12: "1": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1104 column 16: "$t0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1131 column 16: "$s0": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1283 column 12: "333": operand is of incorrect type
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1328 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1362 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1413 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
+Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1458 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
 
+Processing terminated due to errors.
