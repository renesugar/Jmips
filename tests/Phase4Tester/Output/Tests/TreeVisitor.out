Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 97 column 12: "100000000": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 170 column 12: "100000000": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 178 column 12: "50000000": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 189 column 12: "100000000": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 294 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 313 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 328 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 382 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 423 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 438 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 454 column 16: "$t1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 459 column 16: "$t2": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 460 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 513 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 642 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 670 column 16: "$s1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 708 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 719 column 12: "1": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1104 column 16: "$t0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1131 column 16: "$s0": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1283 column 12: "333": operand is of incorrect type
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1328 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1362 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1413 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)
Error in /Users/krishna/Desktop/Computer Science/BS179/test_179e/tests/Phase4Tester/./Output/Tests/TreeVisitor.compiler.out line 1458 column 3: "la": Too few or incorrectly formatted operands. Expected: la $t1,($t2)

Processing terminated due to errors.
